test/typescript-version.test.ts

```diff
@@ -31,9 +32,10 @@ describe('typescript version', () => {
   });
 
   it('the version in the plop file matches the root version', () => {
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
     const rootPackageJSON = require('../package.json');
```

Comment:
Also not sure this rule is useful for tests, does it get mad about `require.requireActual` / `require.requireMock`?